tests/experiment/test_model.py

```diff
@@ -84,17 +84,17 @@ def test_from_pars(self):
         assert mod._resolution_function([100]) == 2.0
 
     def test_add_item(self):
-        m1 = Material.from_pars(6.908, -0.278, 'Boron')
-        m2 = Material.from_pars(0.487, 0.000, 'Potassium')
-        l1 = Layer.from_pars(m1, 5.0, 2.0, 'thinBoron')
-        l2 = Layer.from_pars(m2, 50.0, 1.0, 'thickPotassium')
-        ls1 = LayerCollection.from_pars(l1, l2, name='twoLayer1')
-        ls2 = LayerCollection.from_pars(l2, l1, name='twoLayer2')
-        o1 = RepeatingMultilayer.from_pars(ls1, 2.0, 'twoLayerItem1')
-        o2 = RepeatingMultilayer.from_pars(ls2, 1.0, 'oneLayerItem2')
-        surfactant = SurfactantLayer.default()
-        multilayer = Multilayer.default()
-        d = Sample.from_pars(o1, name='myModel')
+        m1 = Material(6.908, -0.278, 'Boron')
```

Comment:
please consider testing the new ways of instantiating `Material` (and other objects) with parameters and with None.
The tests here only use positional arguments and `Number`s.

Reply:
I have added new tests for `material` and `layer`.